### ISSUE TITLE
False positives: incorrectly indented expression body functions with `WRONG_INDENTATION` inspection

### ISSUE THREAD

**Issue by akuleshov7** [saveourtool/diktat#1330]
```kotlin
    @Test
    fun `checking that suppression with ignore everything works`() {
        val code =
                """
                    @Suppress("diktat")
                    fun foo() {
                        val a = 1
                    }
                """.trimIndent()
        lintMethod(code)
    }
```

<img width="1205" alt="image" src="https://user-images.githubusercontent.com/58667063/171014226-54696a6a-319d-42e8-9b14-b74fd30ee8f5.png">

```
Warn (2:1) [WRONG_INDENTATION] only spaces are allowed for indentation and each indentation should equal to 4 spaces (tabs are not allowed): expected 0 but was 4
Warn (3:1) [WRONG_INDENTATION] only spaces are allowed for indentation and each indentation should equal to 4 spaces (tabs are not allowed): expected 4 but was 8
Warn (10:1) [WRONG_INDENTATION] only spaces are allowed for indentation and each indentation should equal to 4 spaces (tabs are not allowed): expected 4 but was 8
Warn (11:1) [WRONG_INDENTATION] only spaces are allowed for indentation and each indentation should equal to 4 spaces (tabs are not allowed): expected 0 but was 4
Warn (11:5) [WRONG_INDENTATION] only spaces are allowed for indentation and each indentation should equal to 4 spaces (tabs are not allowed): no newline at the end of file demo-1874f666-bfd2-48f0-a471-384a98d6af37.kt
```

## Environment information
* diktat version: 1.1.0
* build tool (maven/gradle): maven
* how is diktat run (CLI, plugin, etc.): plugin
* kotlin version: 1.6
* operating system: MAC

**Comment by akuleshov7** [comment]
@0x6675636b796f75676974687562 please check it ASAP

**Comment by 0x6675636b796f75676974687562** [comment]
Also:

```kotlin
fun foo() =
        println() // <- 4 spaces (1 tab indent)
```

**Comment by 0x6675636b796f75676974687562** [comment]
The problem from the original description fixed as #1347.

None of the issues with mis-indented expression bodies (from both [this](https://github.com/saveourtool/diktat/issues/1330#issuecomment-1142388750) and [this](https://github.com/saveourtool/diktat/issues/1312#issuecomment-1142337550) comment) was reproduced.

Closing.
